### PR TITLE
[VI] Add `Modding/Bubble`

### PR DESCRIPTION
**Pull request by MineTurtlee**
It has maybe, been a week since then and there I think is NO REVIEWER! Help here!

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] Finished all tasks (Continous Integration)

### CODE REVIEW

File: wiki/Modding/Bubble/vi.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Bubble
+
+Cái **bubble** (![Bubble icon](/wiki/shared/icon/bubble.gif)) là một biểu tượng bài đăng trên diễn đàn được sử dụng bởi các thành viên của các nhóm modding sau đây tham gia vào [Quá trình xếp hạng beatmap](/wiki/Beatmap_ranking_procedure):
+
+- [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*)
+- [Mapping Assistance Team](/wiki/People/Mapping_Assistance_Team) (*MAT*)
+- [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (*BN*)
+- [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) (*QAT*)
+
+Là một bước tiếp theo của [Hệ thống ranking cũ](/wiki/Modding/Forum_modding) sau [Proto-bubble](/wiki/Modding/Proto-bubble), một bubble bình thường có nghĩa là một [beatmap](/wiki/beatmap) đã chuẩn bị được xếp hạng từ ý kiến của modder. Những beatmap được bubble sẽ được kiểm tra bởi một thành viên khác của BAT hoạc BN và được [xếp hạng](/wiki/beatmap/category#ranked) hay [chấp nhận](/wiki/beatmap/category#approved) miễn là chỉ có những thay đổi nhỏ cho beatmap là bắt buộc.
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:16]:
```suggestion
Là một bước tiếp theo của [Hệ thống ranking cũ](/wiki/Modding/Forum_modding) sau [Proto-bubble](/wiki/Modding/Proto-bubble), một bubble bình thường có nghĩa là một [beatmap](/wiki/Beatmap) đã chuẩn bị được xếp hạng từ ý kiến của modder. Những beatmap được bubble sẽ được kiểm tra bởi một thành viên khác của BAT hoạc BN và được [xếp hạng](/wiki/Beatmap/category#ranked) hay [chấp nhận](/wiki/Beatmap/category#approved) miễn là chỉ có những thay đổi nhỏ cho beatmap là bắt buộc.
```

File: wiki/Modding/Bubble/vi.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ 
+Trong [quy trình xếp hạng beatmap](wwiki/Beatmap_Ranking_Procedure) hiện , cái tương đương với bubble là [đề cử](/wiki/beatmap_ranking_procedure#nominations) được cho bởi một [người đề cử beatmap](/wiki/people/beatmap_nominators).
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:18]:
```suggestion
Trong [quy trình xếp hạng beatmap](wiki/Beatmap_ranking_procedure) hiện , cái tương đương với bubble là [đề cử](/wiki/Beatmap_ranking_procedure#nominations) được cho bởi một [người đề cử beatmap](/wiki/people/Beatmap_Nominators).
```

File: wiki/Modding/Bubble/vi.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ 
+## Lịch sử
+ 
+*Xem thêm [Dòng thời gian mapping và modding](/wiki/History_of_osu!/Mapping_And_Modding_Timeline)*
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:22]:
```suggestion
*Xem thêm [Dòng thời gian mapping và modding](/wiki/History_of_osu!/Mapping_and_Modding_Timeline)*
```

File: wiki/Modding/Bubble/vi.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ 
+Bubble được giới thiệu bởi ::{ flag=AU }:: [peppy](/wiki/People/peppy) vào ngày 29 tháng 10 năm 2007, cho "những beatmap đang được xem xét cho lượt chơi đã được xếp hạng (đang chờ phản hồi xa hơn bởi mod)". Đặt một biểu tường cho thread của beatmap sang một cái bubble là một cách cho thành viên BAT để chỉ ra rằng beatmap có chất lượng tốt và đã theo [tiêu chí xếp hạng](/wiki/Ranking_Criteria)<!-- internal reference: https://osu.ppy.sh/community/forums/topics/619 -->.
+
+Vào ngày 3 tháng 10 năm 2010, Nhóm MAT đã nhận được quyền để sử dụng biểu tượng bubble, cùng với BAT!
+
+Sau [hệ thống thảo luận beatmap](/wiki/beatmap_discussion) đã được hoàn toàn thêm vào và đã hóa thành giao diện chính cho việc modding vào tháng 11 năm 2017, quản lý beatmap dựa trên diễn đàn đã bị loại bỏ dần, với bubble được thay bằng đề cử.
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:28]:
```suggestion
Sau [hệ thống thảo luận beatmap](/wiki/Beatmap_discussion) đã được hoàn toàn thêm vào và đã hóa thành giao diện chính cho việc modding vào tháng 11 năm 2017, quản lý beatmap dựa trên diễn đàn đã bị loại bỏ dần, với bubble được thay bằng đề cử.
```

File: wiki/Modding/Bubble/vi.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Cơ chế
+
+::: Infobox
+![](img/list-of-bubbles.png "Danh sách những beatmap được bubble, cùng với những bubble được nổ và những map được đánh dấu sao")
+:::
+
+Thread của những beatmap được bubble đã được liệt kê trên những trang đầu tiên của [Forum những beatmap đang chờ](https://osu.ppy.sh/community/forums/6), đã được sắp xếp bởi [sự ưu tiên sao](/wiki/modding/star_piority) của chúng và được hiệu lực theo vai những chuỗi forum.
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:36]:
```suggestion
Thread của những beatmap được bubble đã được liệt kê trên những trang đầu tiên của [Forum những beatmap đang chờ](https://osu.ppy.sh/community/forums/6), đã được sắp xếp bởi [sự ưu tiên sao](/wiki/modding/Star_priority) của chúng và được hiệu lực theo vai những chuỗi forum.
```

File: wiki/Modding/Bubble/vi.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+### Yêu cầu
+
+- Để được bubble, [sự ưu tiên sao](/wiki/modding/star-piority) của một beatmap phải có ít nhất là 8.
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:40]:
```suggestion
- Để được bubble, [sự ưu tiên sao](/wiki/modding/Star_priority) của một beatmap phải có ít nhất là 8.
```
Just going to mention that I can't see any reason for `/wiki/modding/Star_priority` to be linked twice and also considering that they are used within the same subject matter.

File: wiki/Modding/Bubble/vi.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- Yêu cầu cho một beatmap để được xem xét cho loại [xếp hạng](/wiki/beatmap/category#ranked) là có một bubble.
```
Comment (WindowsMeosu) [inline wiki/Modding/Bubble/vi.md:41]:
```suggestion
- Yêu cầu cho một beatmap để được xem xét cho loại [xếp hạng](/wiki/Beatmap/Category#ranked) là có một bubble.
```

**Review by WindowsMeosu — CHANGES_REQUESTED** [review]
Friendly reminder that links are case-sensitive.

**Comment by MineTurtlee** [comment]
> Friendly reminder that links are case-sensitive.

Thanks for the friendly reminder! I will remember it when I do more contributions!

**Review by MineSkamaxx — DISMISSED** [review]
I think this is good to go. (Mainly no errors)

**Comment by MineTurtlee** [comment]
I think I won't reopening that pr.

**Comment by MineTurtlee** [comment]
> another friendly reminder
> 
> [#9365 (comment)](https://github.com/ppy/osu-wiki/pull/9365#issuecomment-1546661598)
> 
> image

Hmm, how? (Custom branches??)

**Review by MineTurtlee — COMMENTED** [review]
Well I think I shall make no errors anytime  I open a pr. :)

**Comment by TicClick** [comment]
going to merge your PR for now, but please either [make less commits, batching multiple changes](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits), or disable the `master` branch protection which prevents me from compacting your changes

also please see https://github.com/ppy/osu-wiki/pull/9366#issuecomment-1548289071 -- a maintainers will hit the `Update` button when necessary

File: wiki/Modding/Bubble/vi.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+---
+tags:
+ - bubble pop
+ - full bubble
+---
```
Comment (TicClick) [inline wiki/Modding/Bubble/vi.md:5]:
```suggestion
no_native_review: true
---
```

**Review by TicClick — CHANGES_REQUESTED** [review]
additionally, I can't make ANY changes at all due to the `master` branch protection -- please:
1) accept the suggestion since your translation received no native review
2) disable the branch protection https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#deleting-a-branch-protection-rule
3) alternatively, scope your changes to individual, new branches (no. 2 of https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)

**Comment by MineTurtlee** [comment]
> additionally, I can't make ANY changes at all due to the `master` branch protection -- please:
> 
> 1. accept the suggestion since your translation received no native review
> 2. disable the branch protection https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#deleting-a-branch-protection-rule
> 3. alternatively, scope your changes to individual, new branches (no. 2 of https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)

Okay then.

**Review by MineTurtlee — COMMENTED** [review]
Is this good now?

File: wiki/Modding/Bubble/vi.md (hunk 8 above)
Comment (MineTurtlee) [reply wiki/Modding/Bubble/vi.md:6]: I think that this is done. But I will have to learn more about GitHub (just more, help.)

**Comment by MineTurtlee** [comment]
``` suggestion
hi
```

File: wiki/Modding/Bubble/vi.md
Comment (MineTurtlee) [inline wiki/Modding/Bubble/vi.md:1]: Why doesn't it mark as done?

Comment (WindowsMeosu) [reply wiki/Modding/Bubble/vi.md:1]: To answer your question, changes have been requested yet hadn't been resolved thus merging is blocked until said changes had been fulfilled.

**Review by TicClick — APPROVED** [review]

**Comment by MineTurtlee** [comment]
YAYY! Thank you, it was my first pull on GitHub!

**Pull request merged**